### PR TITLE
СС-18569 Docs update - Switch from Yarn to NPM v8

### PR DESCRIPTION
**Pull request by danilmoro3**
Ticket: https://spryker.atlassian.net/browse/CC-18568

### CODE REVIEW

File: docs/cloud/dev/spryker-cloud-commerce-os/configuring-deployment-pipelines/upcoming-release/configuring-github-actions.md [hunk 1]
```diff
@@ -0,0 +1,424 @@
...
+                  sudo sysctl -w vm.max_map_count=262144
+            - name: Runs Elasticsearch
+              uses: elastic/elastic-github-actions/elasticsearch@master
+              with:
+                  stack-version: 7.6.0
+                  port: 10005
+
+            - uses: actions/setup-node@v1
+              with:
+                  node-version: '12'
```
Comment (FRITS-Kh) [inline docs/cloud/dev/spryker-cloud-commerce-os/configuring-deployment-pipelines/upcoming-release/configuring-github-actions.md:75]:
```suggestion
                  node-version: '16'
```

File: docs/cloud/dev/spryker-cloud-commerce-os/configuring-deployment-pipelines/upcoming-release/configuring-github-actions.md [hunk 2]
```diff
@@ -0,0 +1,424 @@
...
+
+            - name: Generate rest API dcoumentation
+              run: vendor/bin/console rest-api:generate:documentation
+
+            - name: Frontend install-dependencies
+              run: |
+                  vendor/bin/console frontend:project:install-dependencies
+            - name: Install NPM packages
+              run: |
+                  sudo npm install -g --unsafe-perm speccy@0.11.0
```
Comment (FRITS-Kh) [inline docs/cloud/dev/spryker-cloud-commerce-os/configuring-deployment-pipelines/upcoming-release/configuring-github-actions.md:137]:
```suggestion
                  sudo npm install --location=global --unsafe-perm speccy@0.11.0
```

File: docs/marketplace/dev/feature-integration-guides/upcoming-release/marketplace-merchant-portal-core-feature-integration.md [hunk 3]
```diff
@@ -0,0 +1,1275 @@
...
+**.npmrc**
+
+```text
+legacy-peer-deps=true
+```
+
+Run commands from the root of the project:
+
+```bash
+npm i -g @angular/cli@12.2.16
```
Comment (FRITS-Kh) [inline docs/marketplace/dev/feature-integration-guides/upcoming-release/marketplace-merchant-portal-core-feature-integration.md:738]:
```suggestion
npm i --location=global @angular/cli@12.2.16
```

**Review by FRITS-Kh — CHANGES_REQUESTED** [review]

**Review by FRITS-Kh — APPROVED** [review]

**Comment by danilmoro3** [comment]
Do not merge until the release 
`upcoming-release` folder should be renamed

**Review by gibmirmilch — DISMISSED** [review]

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**